hygieia-jenkins-plugin/src/main/java/hygieia/builder/SonarBuilder.java

```diff
@@ -33,8 +36,12 @@
      */
     public static final String URL_PATTERN_IN_LOGS = ".*" + Pattern.quote("ANALYSIS SUCCESSFUL, you can browse ") + "(.*)";
     public static final String URL_PROJECT_ID_FRAGMENT = "/api/projects?format=json&key=%s";
-    public static final String URL_METRIC_FRAGMENT = "/api/resources?format=json&resource=%s&metrics=%s&includealerts=true";
-    public static final String METRICS = "security-violations,ncloc,violations,critical_violations,major_violations,blocker_violations,violations_density,tests,test_success_density,test_errors,test_failures,coverage,line_coverage,sqale_index";
+
+    public static final String CE_URL_PATTERN_IN_LOGS = ".*" + Pattern.quote("More about the report processing at ") + "(.*)";
+    public static final String CE_URL_PROJECT_ID_FRAGMENT = "/ce/task?id=%s";
+
+    public static final String URL_METRIC_FRAGMENT = "/api/resources?format=json&resource=%s&metrics=%s&includealerts=true&includetrends=true";
+    public static final String METRICS = "security-violations,ncloc,violations,critical_violations,major_violations,blocker_violations,violations_density,tests,test_success_density,test_errors,test_failures,coverage,line_coverage,sqale_index,new_violations,new_blocker_violations,new_critical_violations,new_major_violations,new_coverage,new_lines_to_cover,new_line_coverage";
```

Comment:
Lets capture the trend too. Example: http://nemo.sonarsource.org/api/resources?resource=org.codehaus.sonar:sonar&metrics=new_coverage&includetrends=true